SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.WindowsDesktop/Utils/StackTraceHelper.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.AppCenter.Crashes.Utils
+{
+    public class StackTraceHelper
+    {
+        // Exceptions don't always have complete stack traces, so they must be augmented.
+        public static string GenerateFullStackTrace(Exception e)
+        {
+            if (string.IsNullOrEmpty(e.StackTrace))
+            {
+                return e.StackTrace;
+            }
+            var exceptionStackTrace = new System.Diagnostics.StackTrace(e, true);
+
+            // Generate current stack trace. Skip three frames to avoid showing SDK code.
+            var currentStackTrace = new System.Diagnostics.StackTrace(3, true);
```

Comment:
Can we make a note of what the three frames skipped are?  Right now it looks like a magic number